addons/danger/functions/fnc_tacticsAssess.sqf

```diff
@@ -20,12 +20,14 @@
 #define TACTICS_ASSAULT 3
 #define TACTICS_SUPPRESS 4
 #define TACTICS_ATTACK 5
+#define RANGE_MID 220
+#define RANGE_LONG 300
+#define RANGE_THREAT 450
```

Comment:
These should have been in before.  Through they could be transitioned to settings. I propose simply Defining for now. 
RANGE_THREAT points to both armored vehicles and aircraft.

Reply:
if we use Defines in Multiple Areas we should move them to the script_component.hpp in the highest common level

Reply:
Good point. Tempting to do so now, but I feel this pull request is already dangerously sprawling.

Reply:
In fact: What about adding CBA sliders? Allowing customization.  Much akin to the CQB variable.  Something for 2.5+ methinks.